scripts/start

```diff
@@ -1,7 +1,6 @@
 #!/bin/bash
-
 docker network inspect mine-support &>/dev/null || docker network create mine-support
-
 docker-compose down -v
```

Comment:
the -f file specifications are missing from the down command, this will prevent volumes specified in the test override file from being deleted

Reply:
Updated